notebooks/community/vertex_endpoints/optimized_tensorflow_runtime/t5x_base_optimized_online_prediction.ipynb

```diff
@@ -0,0 +1,1551 @@
+{
```

Comment:
<h1>Deploy model to Vertex AI Prediction -&gt;</h1><h1>Deploy model to Vertex AI Endpoint</h1><h1><br></h1>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Done</p>